doc/content/the-things-stack/management/purge/_index.md

```diff
@@ -1,16 +1,93 @@
 ---
-title: "Purging Entities"
+title: "Deleting, Restoring and Purging Entities"
 description: ""
 weight:
 aliases: [/reference/purge]
 ---
 
 When entities are deleted, {{% tts %}} [retains some IDs]({{< ref "reference/id-eui-constraints" >}}) by default to prevent another user re-registering the ID and obtaining historical data.
 
-It is also possible for administrators to purge IDs so that they may be reused. This reference contains information about how to do this, and the implications of purging IDs.
+When entities are deleted, they can be restored within a limited time period. It is also possible for administrators to purge IDs so that they may be reused.
```

Comment:
A bit of repetition at the beginning of both sentences, wouldn't it be appropriate to set everything under a single paragraph.

```
When entities are deleted, {{% tts %}} [retains some IDs]({{< ref "reference/id-eui-constraints" >}}) by default to prevent another user re-registering the ID and obtaining historical data. While soft-deleted an admin can either restore an entity or purge it in order to allow its ID to be reused.
```

Followed by an observation explaining that the restore procedure has to be done within a configurable time frame.